.github/dependabot.yml

```diff
@@ -0,0 +1,49 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    # Workflow files stored in the
+    # default location of `.github/workflows`
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    commit-message:
+      prefix: ":seedling:"
+    labels:
+      - "ok-to-test"
+
+  # Maintain dependencies for go
+  - package-ecosystem: "gomod"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    labels:
+      - "ok-to-test"
```

Comment:
Will this work for go dependencies used by the testdata? When I bumped ginkgo, `make test` failed until I ran `make generate`?

Reply:
The label only allows the tests be executed. Just that.
Also, note that this action is just to open PR's with updates in the github actions. So, it has no relation with make generate at all. The make generate re-generate  the samples projects scaffold under testdata to ensure that the scaffolds matches with the latest changes (it only fails/or is required when you do prs to change the scaffolds)

Reply:
Ok. My thought was if dependabot for example bumps ginkgo (~`go get -u github.com/onsi/ginkgo/v2`) and makes a pr, then GH actions will automatically run `make test` and it would fail. But this isn't a big worry for me and I could be wrong here.